osu.Game/Rulesets/UI/Playfield.cs

```diff
@@ -325,6 +340,8 @@ DrawableHitObject IPooledHitObjectProvider.GetPooledDrawableRepresentation(HitOb
                 // This is done before Apply() so that the state is updated once when the hitobject is applied.
                 if (!dho.IsLoaded)
```

Comment:
So, if this comment is misleading as if the DHO is added and removed from the `Playfield` in one frame then this `if` clause can run multiple times.